dockstore-client/src/main/java/io/dockstore/client/cli/nested/AbstractEntryClient.java

```diff
@@ -161,6 +186,11 @@ public void printGeneralHelp() {
         out("  " + LAUNCH + "           :  launch " + getEntryType() + "s (locally)");
         out("");
         out("  " + DOWNLOAD + "         :  download " + getEntryType() + "s to the local directory");
+        if (getEntryType().toLowerCase().equals(WORKFLOW)) {
```

Comment:
I would do `WORKFLOW.equalsIgnoreCase(getEntryType())`.

1.You know `WORKFLOW` cannot be null
2. Right now this code assumes the value of `WORKFLOW` is lower-case, which is not immediately apparent.

Reply:
Using your suggested code now